adhocracy-plus/assets/scss/_form.scss

```diff
@@ -77,6 +77,7 @@ div.cke_focus {
 .form-check,
 .form-group {
     .django-ckeditor-widget,
+    .btn:not([type="radio"], [type="checkbox"]),
```

Comment:
not sure if type=radio and type=checkbox is needed here, but that is what I saw is used on stage.